news/2025/2025-05-06-expert-global-taiko-showdown-2025-concludes.md

```diff
@@ -0,0 +1,676 @@
+
+Three more consecutive points were tipped towards ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) with mostly razor-thin margins, save for a commendable 90k lead on HR2. At this point, all of a sudden, he was one point away from resetting the bracket.
+
+::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073)'s final pick was between one of two incredibly painful maps: NM5, a chaotic polyrhythmic experience, and DT1, a speedy accuracy nightmare. His choice fell upon NM5, but no one could've anticipated the back-and-forth that ensued.
+
+For the first and only time in the history of EGTS, players and viewers alike would be treated to a true world premiere: a genuine Grand Finals tiebreaker; an 11-minute gauntlet at the end of a nerve-wracking one-hour match to decide whether ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) would be crowned the final EGTS champion, or whether he would have to prove himself all over again. By the end, ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) won with a 47,159-score difference.
+
+Both players were given a desperately needed 10-minute break, as the bracket was reset.
+
+---
+
+The new match started yet again with a show of force in their own picks, yet the advantages didn't make for easy wins. Neither player willing to give up, both continued to flaunt their ambitions at the champion title.
+
+For example, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found a 1.042 *million* HD score on FM3 to win the point, yet it would be impossible to overstate ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s FC with **HR** on his opponents reading- and accuracy-heavy pick.
+
+Both continued to not give away any of their own picks, but it was apparent that something had changed in ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073)'s mind. A 2x100 HD FC on FM1, a *halving* of his previous 100 miss count on DT2 (albeit still losing the pick), and his **near SS** on HD2 showed the audience just what kind of determination he had brought to the match that he wasn't willing to give up on.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/egts-2025-grape_tea-vs-kanten_07.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+However, everyone began to worry that all of this might not have been enough, as the scoreline was still tied at 5–5. The smell of another potential tiebreaker lingered in the air.
```

Comment:
```suggestion
However, everyone began to worry that all of this might not have been enough, as the scoreline was still tied at 5–5. The smell of another potential tiebreaker lingered in the air.
```